system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_endpoint_test.clj

```diff
@@ -75,6 +75,18 @@
                             :Term "SURFACE RADIATIVE PROPERTIES"
                             :VariableLevel1 "REFLECTANCE"}}
             400
+            ["object has missing required properties ([\"concept-ids\"])"]
+
+            "All concept-ids"
+            {:concept-ids ["all" ]
```

Comment:
You test the blank spaces elsewhere. Should get rid of the blank space here.

Reply:
ok. done.